include/xframe/xaxis.hpp

```diff
@@ -87,7 +90,6 @@ namespace xf
         template <class InputIt>
         xaxis(InputIt first, InputIt last);
 
-        const label_list& labels() const noexcept;
         bool is_sorted() const noexcept;
 
         bool empty() const noexcept;
```

Comment:
See the comment in `xaxis_default.hpp`, thses methods can be defined in `xaxis_base.hpp`